src/wizard/monster-info-spoiler.cpp

```diff
@@ -203,32 +200,22 @@ SpoilerOutputResultType spoil_mon_info(concptr fname)
             spoil_out("[N] ");
         }
 
-        sprintf(buf, _("%s/%s  (", "%s%s ("), r_ptr->name.data(), _(r_ptr->E_name.data(), "")); /* ---)--- */
-        spoil_out(buf);
+        spoil_out(std::string(format(_("%s/%s  (", "%s%s ("), r_ptr->name.data(), _(r_ptr->E_name.data(), ""))).data()); /* ---)--- */
```

Comment:
Since the process of calling fromat is not done in spoil_out, shouldn't it be ok to not wrap it in std::string().data()?

(I may have to add .data() when I make the format function return a std::string, but even then std::string() would not be needed.)

Reply:
Since none of the existing uses of spoil_out() were directly passed the result of format(), I wrapped format() with std::string() as a way of protecting it against some future change that would have spoil_out() call format() internally.  With your planned change to have format() return a std::string, I can remove that wrapping.